public/sw.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
The file at location `/${data.service}/articles/sw.js` is being registered in the `client.js`. This file should be moved into the relevant `/public/news/` directory.